.ci/build_windows.bat

```diff
@@ -43,34 +29,29 @@ for /f "tokens=* USEBACKQ" %%g in (`python "scripts\mumble-version.py" --format
 :: in that case the variable substitution of that variable in the expression below fails (is replaced with empty string)
 :: Also we can't anything else inside the if body as this will cause the curl command to always be executed.
 if defined MUMBLE_BUILD_NUMBER_TOKEN (
-	for /f "tokens=* USEBACKQ" %%g in (`curl "https://mumble.info/get-build-number?commit=%BUILD_SOURCEVERSION%&version=%VERSION%&token=%MUMBLE_BUILD_NUMBER_TOKEN%"`) do (set "BUILD_NUMBER=%%g")
+	for /f "tokens=* USEBACKQ" %%g in (`curl "https://mumble.info/get-build-number?commit=%APPVEYOR_REPO_COMMIT%&version=%VERSION%&token=%MUMBLE_BUILD_NUMBER_TOKEN%"`) do (set "BUILD_NUMBER=%%g")
```

Comment:
I guess the `APPVEYOR_REPO_COMMIT` variable should be renamed as well (and mapped by the YAML files)

Reply:
Yes, indeed, I missed many things (as you can see from the many force-pushes)...